ppa/archive/eebo_tcp.py

```diff
@@ -27,16 +27,16 @@ def short_id(volume_id):
 P5_TAG = TagNames(**{tag: "{%s}%s" % (TEI_NAMESPACE, tag) for tag in _p5_tags})
 
 
-class TeiXmlObject(xmlmap.XmlObject):
+class TeiXmlObject(core.XmlObject):
     ROOT_NAMESPACES = {"t": TEI_NAMESPACE}
 
 
 class MixedText(TeiXmlObject):
     divider = "∣"
 
     # count following notes as a quick check to bail out of note detection logic
-    # (can't use eulxml boolean field here because it assumes string values for true/false)
-    has_notes = xmlmap.IntegerField("count(following::NOTE)")
```

Comment:
If I'd thought about how much the code was going to change based on the revision to the imports I might have changed my answer to how you changed this in neuxml!  (I recognize it's kind of late now).  This feels much less readable/obvious to me (although I'm also definitely biased after using the old syntax for so long).

LMK if you have any thoughts; how hard to support both with explicit / non-\*  imports for existing fields?